### PR TITLE
misc fixes

### PR DESCRIPTION
**Pull request by dustinreed-info**

### CODE REVIEW

File: alpaca_data_cleaned.json [hunk 1]
```diff
@@ -17570,9 +17570,9 @@
         "output": "o"
     },
     {
-        "instruction": "Create a task that a GPT model can complete with a given input and output.",
-        "input": "",
-        "output": "Instruction: Given the following input, generate a list of three adjectives that describe the input:\nInput: Halloween\n\nOutput: Spooky, eerie, macabre."
+        "instruction": "Instruction: Given the following input, generate a list of three adjectives that describe the input.",
```
Comment (niclimcy) [inline alpaca_data_cleaned.json:17573]: Should there even be a "instruction:" within the instruction parameter?

File: alpaca_data_cleaned.json [hunk 2]
```diff
@@ -191217,7 +191217,7 @@
     {
         "instruction": "Determine the median number from the following list: 3, 85, 6, 7, 1",
         "input": "3, 85, 6, 7, 1",
-        "output": "6"
+        "output": "20.4"
```
Comment (niclimcy) [inline alpaca_data_cleaned.json]: Original output was correct, median is not the same as mean

File: alpaca_data_cleaned.json [hunk 3]
```diff
@@ -232287,7 +232287,7 @@
     {
         "instruction": "Divide the equation below and simplify.",
         "input": "3x + 6y = 15",
-        "output": "x = 5 - 2y"
+        "output": "x + 2y = 5"
```
Comment (niclimcy) [inline alpaca_data_cleaned.json]: Original output is technically more simplified

File: alpaca_data_cleaned.json [hunk 4]
```diff
@@ -222082,7 +222082,7 @@
     {
         "instruction": "Compute the mean and standard deviation for this set of numbers.",
         "input": "2, 3, 7, 5",
-        "output": "Mean = 4.5, Standard deviation = 2.121320344"
+        "output": "Mean = 4.25, Standard deviation = 1.8306"
```
Comment (niclimcy) [inline alpaca_data_cleaned.json:222085]: Sample standard deviation should be 2.217355783 (as calculated by my scientific calculator). Fun fact, chatgpt3 got this question wrong too

File: alpaca_data_cleaned.json [hunk 5]
```diff
@@ -234642,7 +234642,7 @@
     {
         "instruction": "Write an expression for the given mathematical equation.",
```
Comment (niclimcy) [inline alpaca_data_cleaned.json:234643]: The phrasing of some math instructions are weird for basic calculations

File: alpaca_data_cleaned.json [hunk 6]
```diff
@@ -247407,7 +247407,7 @@
     {
         "instruction": "Calculate the Standard Deviation of the following set of data:",
         "input": "14, 18, 12, 16, 16, 20",
-        "output": "The Standard Deviation of the given set of data is 3.27."
+        "output": "The Standard Deviation of the given set of data is approximately 2.5819."
```
Comment (niclimcy) [inline alpaca_data_cleaned.json:247410]: ~~Wrong answer again, this is population standard deviation, sample standard deviation is approximately 2.82.~~

Answer is correct and should be population SD, however we should still consider the following:

Also we might want to consider standardising the accuracy of every math approximation to either 3 sig fig or 2 decimal places

File: alpaca_data_cleaned.json (hunk 4 above)
Comment (niclimcy) [reply alpaca_data_cleaned.json:222085]: Sorry it should be population standard deviation as the output given that the instruction does not specify where those numbers come from. This answer is still wrong though, population SD = 1.920286437

**Comment by dustinreed-info** [comment]
@nicknitewolf Hey I made the requested corrections, thanks for the second set of eyes!

**Comment by niclimcy** [comment]
Looks good now, but I bet there are other bad math in this data set. GPT3 is notoriously bad at math

**Review by gururise — APPROVED** [review]
Thanks for the PR!

**Pull request merged**